### PR TITLE
Added page on indexing ethereum local node

### PR DESCRIPTION
**Pull request by RaekwonIII**
Restructured the order of pages in Tutorial
Added missing page haders where needed

### CODE REVIEW

File: docs/tutorials/ethereum-local-development.md [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+### Archive Docker image
+
+In order to index the local Ethereum node, we need to use a pre-built Docker image of Subsquid's Ethereum Archive. You can do it, by creating a `docker-compose.archive.yml` file, and paste this:
+
+```yaml
+version: "3"
+
+services:
+  worker:
+    image: massimocodes/eth-archive-worker:latest
```
Comment (dzhelezov) [inline docs/tutorials/ethereum-local-development.md:211]: We should use an official subsquid image here

File: docs/tutorials/ethereum-local-development.md [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+            "--http-req-concurrency", "10",
+            "--best-block-offset", "10",
+            "--rpc-urls", "http://host.docker.internal:8545/",
+            "--max-resp-body-size", "30",
+            "--resp-time-limit", "5000",
+            "--query-concurrency", "16",
+    ]
+    volumes:
+      - db:/data/db
+      - parquet:/data/parquet
```
Comment (dzhelezov) [inline docs/tutorials/ethereum-local-development.md:233]: I think Ozgur has prepared a version when no parquet is needed

**Review by dzhelezov — CHANGES_REQUESTED** [review]

File: docs/tutorials/ethereum-local-development.md (hunk 1 above)
Comment (RaekwonIII) [reply docs/tutorials/ethereum-local-development.md:211]: I know, but they are not public. I don't have access to them, or cannot create them.

File: docs/tutorials/ethereum-local-development.md (hunk 2 above)
Comment (RaekwonIII) [reply docs/tutorials/ethereum-local-development.md:233]: It shouldn't be needed, but the archive won't start without the option `"--data-path"`:

```bash
ethereum-local-indexing-worker-1  | error: The following required arguments were not provided:
ethereum-local-indexing-worker-1  |   --data-path <DATA_PATH>
ethereum-local-indexing-worker-1  | 
ethereum-local-indexing-worker-1  | Usage: eth-archive-worker --db-path <DB_PATH> --data-path <DATA_PATH> --request-timeout-secs <REQUEST_TIMEOUT_SECS> --connect-timeout-ms <CONNECT_TIMEOUT_MS> --block-batch-size <BLOCK_BATCH_SIZE> --http-req-concurrency <HTTP_REQ_CONCURRENCY> --max-resp-body-size <MAX_RESP_BODY_SIZE> --resp-time-limit <RESP_TIME_LIMIT> --server-addr <SERVER_ADDR> --best-block-offset <BEST_BLOCK_OFFSET> --rpc-urls <RPC_URLS> --query-concurrency <QUERY_CONCURRENCY>
ethereum-local-indexing-worker-1  | 
ethereum-local-indexing-worker-1  | For more information try '--help'
ethereum-local-indexing-worker-1 exited with code 2
```

**Comment by dzhelezov** [comment]
@ozgrakkurt can you please ping here once an official image is published

File: docs/tutorials/ethereum-local-development.md (hunk 2 above)
Comment (dzhelezov) [reply docs/tutorials/ethereum-local-development.md:233]: @ozgrakkurt was this is fixed in the latest release?

Comment (RaekwonIII) [reply docs/tutorials/ethereum-local-development.md:233]: Don't know if you already had a conversation on another channel (Slack), but I'm quoting my own conversation with Ozgur:

> we would have to go to old code branch modify it and then release it for this. Current head is not very usable at the moment

So I guess it is fixed, but in a different branch, so we don't have an image ready, yet.

File: docs/tutorials/ethereum-local-development.md (hunk 1 above)
Comment (RaekwonIII) [reply docs/tutorials/ethereum-local-development.md]: Changed

File: docs/tutorials/ethereum-local-development.md (hunk 2 above)
Comment (RaekwonIII) [reply docs/tutorials/ethereum-local-development.md]: Fixed with latest release

**Comment by RaekwonIII** [comment]
@dzhelezov I have changed to subsquid image and tested it. Ozgur fixed the fact that worker was still demanding for `--data-path` option. So we should be good to go.
Should we bring @abernatskiy into the fold for one last check before merging this?

**Comment by abernatskiy** [comment]
Will go through the tutorial some time this week. Please do not delete the branch yet.

**Pull request merged**